src/utils/style.ts

```diff
@@ -117,29 +123,61 @@ export const cssLeadingToken = ({
   return '';
 };
 
-export const colorVariable = (color: TextColor | undefined): CSSProperties => {
+const PascalToKebab = (str: string) => {
```

Comment:
The functions in this file convert the colour keys specified in the component to CSS Variables.